MBTA_problem_attempt.rb

```diff
@@ -0,0 +1,78 @@
+#calculate the number of stops to park st. then calculate from where park st. appears on the relevant array to the stop.
+
+# red_line_stations = [ "Alewife", "Davis", "Porter", "Harvard", "Central", "Kendell/MIT", "Park Street", "South Station"]
+
+# red_line_stations = {
+#   :alewife => 1,
+#   :davis => 2,
+#   :porter => 3,
+#   :harvard => 4,
+#   :central => 5,
+#   :kendell => 6,
+#   :parkstreet => 7,
+#   :southstation => 8
+# }
+
+# Green_Line = ["Haymarket", "Government Center", "Park Street", "Boylston", "Arlington", "Copley"]
+
+# Orange_Line = ["North Station", "Haymarket", "Park_Street", "Slate Street"]
+
+# def is_on_the_same_line?(origin_line, destination_line)
+#   if origin_line == destination_line
+#     return true
+#   else
+#     return false
+#   end
+# end
+
```

Comment:
It's good that you wanted to break out different parts of the calculation into separate methods.